pkgs/development/libraries/cosmopolitan/default.nix

```
@@ -0,0 +1,57 @@
+{ lib, gcc9Stdenv, fetchFromGitHub, makeWrapper, runCommand, cosmopolitan }:
+
+gcc9Stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+  src = fetchFromGitHub {
+    owner = "jart";
+    repo = "cosmopolitan";
+    rev = "94afa982c35d4322999b5591f6d41d5cfc3f19a5";
+    sha256 = "sha256-LaXkFMoArg5LUw4AdyhOqxzSJLdQCwHVBK2+lXVmxuI=";
+  };
+
+  postPatch = ''
+    sed -i -e "s|/usr/bin/env bash|/bin/sh|" build/findtmp
```

```suggestion
    patchShebangs build/
```
tiny nitpick, that way we should never have to worry about those again. Apart from that LGTM.

For the record, there does not appear to be a `/usr/bin/env` in the sandbox.

---

updated (though now there are some warnings early in the process)

had to modify the wrapper because GCC cares about order of arguments (#85510 ) 

added new test for cosmoc

---

You're correct.

---

should we revert to the sed?